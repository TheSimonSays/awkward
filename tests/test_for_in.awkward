let arr1 = [1, 2, 3, 4, 5];
let arr2 = [1, 2, 30, ["a", "b"]];

for (let i in arr1) {
    print("index is " + i + ", value is " + arr1[i])
}

for (let i in arr2) {
    if (type(arr2[i]) == "array") {
        for(let j in arr2[i]) {
            print(arr2[i][j]);
        }
    } else {
        print(arr2[i]);
    }
}
