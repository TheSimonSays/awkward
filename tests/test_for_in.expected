index is 0, value is 1
index is 1, value is 2
index is 2, value is 3
index is 3, value is 4
index is 4, value is 5
1
2
30
a
b
